wearewondrous/psh-toolbelt
# Tsm 85 large backup s3

# Background
Large File Upload (files>5GB) is not possible with existing implementation. Have to fix this issue and allow large files to be uploaded.

# Changes
Use `MultipartUploader` instead of  `putObject` as Multipart Upload allow files upload regardless of size limitation.
_Refs: https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/s3-multipart-upload.html_

# Testing Instruction:
Integra PR(`integra/pull/214`) is set to check if the command(That is run in a scheduled manner in origin setup) is successfully backing up necessary things. To test this 
- Login to pr env using `platform ssh -e pr-214`
- Run backup Command `vendor/bin/psh-toolbelt backup:branch --force`
- Check if backup is stored in aws bucket using `aws s3 ls s3://wndrs-backups-test --recursive` (or any relevant command) and also configure `aws` client with credentials from Jira Ticket
